### DISCUSSION TITLE
"Full screen" icon on figure images

### DISCUSSION THREAD

**Discussion by mmpadilla15** [thegetty/quire#519] [Get Help] [answered]
Wondering if @lizneely or someone else can advise on this ... by default, the Quire template places the figure number next to the "full screen" icon in figure images, like this:
![Capture2](https://user-images.githubusercontent.com/74433007/194100610-f3b60cc5-692e-43b7-b14a-ac1d7cebfca4.PNG)

It looks like the Georgia O'Keeffe Museum has removed the figure number so all that remains is the "full screen" icon, like this:
![Capture](https://user-images.githubusercontent.com/74433007/194100849-52ef03e8-f79f-4a73-a802-85bca2cc5b5f.PNG)

I'm guessing this is done somewhere is CSS? Can anyone advise on how to do that?

**Comment by Erin-Cecele** [comment, marked as answer]
@mmpadilla15 Are you using the `label` attribute in your `figures.yaml` file or are the labels being automatically generated?

**Reply by mmpadilla15** [reply]
@Erin-Cecele Yes. OMG ... is it as easy as just removing the label???

<img width="527" alt="Screen Shot 2022-10-05 at 11 29 39 AM" src="https://user-images.githubusercontent.com/74433007/194113399-4e021293-2c35-4596-88f4-d20f946f1732.png">

**Reply by Erin-Cecele** [reply]
Yes! If you know you don't want/need them labeled, then that is a lot easier than messing with the CSS.

**Reply by mmpadilla15** [reply]
Whew! @Erin-Cecele , then that makes this easier than I thought it would be! Thanks!

**Reply by lizneely** [reply]
@mmpadilla15 If I did it, you know it had to be an easy fix! :)
